consensus/wal.go

```diff
@@ -95,6 +96,11 @@ func (wal *baseWAL) Group() *auto.Group {
 	return wal.group
 }
 
+func (wal *baseWAL) SetLogger(l log.Logger) {
```

Comment:
Do we need to call this from somewhere?

Reply:
We do call it, already. This method overrides a default one.